src/gurt/debug.c

```diff
@@ -574,16 +574,30 @@ d_log_init_adv(char *log_tag, char *log_file, unsigned int flavor,
 int
 d_log_init(void)
 {
-	char	*log_file;
+	char	*log_file, *log_file_pid_append;
 	int	 flags = DLOG_FLV_LOGPID | DLOG_FLV_FAC | DLOG_FLV_TAG;
 	int	 rc;
+	char     buffer1[20], buffer2[100];
+	pid_t    pid;
 
 	log_file = getenv(D_LOG_FILE_ENV);
 	if (log_file == NULL || strlen(log_file) == 0) {
 		flags |= DLOG_FLV_STDOUT;
 		log_file = NULL;
 	}
 
+	log_file_pid_append = getenv(D_LOG_FILE_APPEND_PID_ENV);
+	if (log_file != NULL && log_file_pid_append != NULL) {
+	        if (strcmp(log_file_pid_append, "0")!=0) {
```

Comment:
(style)  code indent should use tabs where possible
(style)  spaces required around that '!=' (ctx:VxV)